tmpl/blog/announcing-mirage-35-release.md

```diff
@@ -0,0 +1,105 @@
+
+### Noteworthy package updates
+
+- [cstruct 3.6.0](https://github.com/mirage/ocaml-cstruct) API changes, repackaging (see [this announcement](https://discuss.ocaml.org/t/ann-cstruct-3-0-0-with-packaging-changes) and [this announcement](https://discuss.ocaml.org/t/psa-cstruct-3-4-0-removes-old-ocamlfind-subpackage-aliases)
+- [ipaddr 3.0.0](https://github.com/mirage/ocaml-ipaddr) major API changes, the s-expression serialisation is a separate subpackage, macaddr is now a standalone opam package
+- [base64 3.0.0](https://github.com/mirage/base64) performance and API changes, see [this announcement](https://discuss.ocaml.org/t/ann-major-release-of-base64-article)
+- [git 2.0.0](https://github.com/mirage/ocaml-git), read [this announcement](https://discuss.ocaml.org/t/ann-ocaml-git-2-0), as well as [its design and implementation](https://discuss.ocaml.org/t/ocaml-git-git-design-and-implementation)
+- [io-page 2.0.0](https://github.com/mirage/io-page), see [this announcement](https://discuss.ocaml.org/t/ann-io-page-2-0-0-with-packaging-changes)
+- [cohttp 2.0.0](https://github.com/mirage/ocaml-cohttp), see [this announcement](https://discuss.ocaml.org/t/ann-major-releases-of-cohttp-conduit-dns-tcpip)
+- [dns 1.0.0](https://github.com/mirage/ocaml-dns), see [this announcement](https://discuss.ocaml.org/t/ann-major-releases-of-cohttp-conduit-dns-tcpip)
+- [conduit 1.0.0](https://github.com/mirage/ocaml-conduit), see [this announcement](https://discuss.ocaml.org/t/ann-major-releases-of-cohttp-conduit-dns-tcpip)
+
+## More features and bugfixes
+
+- A [httpaf device](https://github.com/mirage/mirage/pull/955) is now part of mirage
+- [libvirt.xml is generated for virtio target](https://github.com/mirage/mirage/pull/903)
+- [Unix target now include -tags thread](https://github.com/mirage/mirage/issues/861) (for mirage-framebuffer SDL support)
+- Various modules (IPv6, DHCP) are explicit about their dependency to the random device
+- [QubesDB can be requested in config.ml when the target is Xen](https://github.com/mirage/mirage/pull/807)
+
+You may also want to read the [MirageOS 3.2.0 announcement](https://discuss.ocaml.org/t/ann-mirage-3-2-0) and the [MirageOS 3.3.0 announcement](https://discuss.ocaml.org/t/mirage-3-3-0-released).
+
+## Next steps
+
+We are working on [further](https://github.com/mirage/mirage/issues/969) [changes](https://github.com/mirage/functoria/pull/167) which revise the `mirage` internal build system to [dune](https://dune.build). At the moment it uses `ocamlbuild`, `ocamlfind`, `pkg-config`, and `make`. The goal of this change is to make MirageOS more developer-friendly. On the horizon we have MirageOS unikernel monorepos, incremental builds, pain-free cross-compilation, documentation generation, ...
```

Comment:
... and an Irmin 2.0 release :-)

Reply:
I pushed 3e96dc4 (ok @samoht? @mato?). we can also just get rid of the next steps section since we don't want to promise too much, but may as well be a nice way to keep people interested. :)